documentation/docs/04-hooks.md

```diff
@@ -125,7 +131,7 @@ export function getSession(event) {
 					email: event.locals.user.email,
 					avatar: event.locals.user.avatar
 				}
-			}
+		  }
```

Comment:
this should be changed back to a tab

Reply:
Not sure what is causing this one, can't find a setting that would do it. Fixed now

Reply:
the code blocks in markdown files aren't linted. If it's just a regular `.js` file it should be handled though

Reply:
This still doesn't look fixed.